src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
+
+    if follower.detect() == False:
+        print('Detect False')
+        follower._speed = follower.LEADER_MAX_SPEED
+        turn = turn + 35
```

Comment:
Why are you adding 35 to turn? What does this mean? You're also not doing anything with turn, so I don't understand what this script is doing?